structure_tests/types/utils.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package types
```

Comment:
not a big deal, but would it make sense for this to be in a different `utils` directory? then you would call `utils.CompileAndRunRegex` instead of `types.CompileAndRunRegex`